.circleci/config.yml

```diff
@@ -178,6 +178,7 @@ jobs:
       USE_GANACHE_DEPLOYMENT_CACHE: true
       GANACHE_CACHE_FOLDER: ../../.ganache-deployments
       GANACHE_PORT: 8547
```

Comment:
@junjizhi `NODE_ENV` is set to `test` in the CircleCI project settings. So, in theory, we don't need to set it here.

Reply:
Yeah. Seems like I couldn't see or change any CircleCI settings for my fork repo builds. I guess all of your development is based on this monorepo, and nobody in your team used the fork-based workflow before?